style.css

```diff
@@ -0,0 +1,635 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap');
+
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  font-family: 'Cocogoose', Lato, sans-serif;
```

Comment:
- [ ] I think numbers are not displaying right with this font style. I suggest you give another fallback fonts/styling so that you won't get undesirable property. 

![image](https://user-images.githubusercontent.com/64894927/132691009-b2a57d54-2241-4df5-96c5-abc82e673ff7.png)
![image](https://user-images.githubusercontent.com/64894927/132691076-1e7d9cf7-5807-4377-87bb-5c3a510b37e4.png)
![image](https://user-images.githubusercontent.com/64894927/132691292-ed77b3f4-56dc-44dd-bd1d-9ec0249fc587.png)